lapce-data/src/terminal.rs

```diff
@@ -447,6 +447,23 @@ impl KeyPressFocus for LapceTerminalViewData {
                         );
                     }
                 }
+                FocusCommand::TerminalCopySelection => {
+                    if self.terminal.mode == Mode::Terminal {
+                        let mut raw = self.terminal.raw.lock();
+                        let term = &mut raw.term;
+                        if let Some(content) = term.selection_to_string() {
+                            Application::global().clipboard().put_string(content);
```

Comment:
There should be a `SystemClipboard` struct somewhere, that could be used to deduplicate these lines.

Reply:
I could, but I would have to create an instance of the struct, I think (unless there already is one, and I can't see it).  
Actually, I might just do it, it helps readability.